### PR TITLE
🤖 Transformers, backend dependabots in disguise

### PR DESCRIPTION
**Pull request by sdevalapurkar-bcgov**
# Description

This PR includes the following proposed change(s):

- 🤖 Transformers, backend dependabots in disguise

## Type of change

- dependencies

### CODE REVIEW

File: src/backend/management-api/pom.xml [hunk 1]
```diff
@@ -103,12 +103,12 @@
     <dependency> 
       <groupId>io.springfox</groupId>  
       <artifactId>springfox-swagger2</artifactId>  
-      <version>2.9.2</version> 
+      <version>3.0.0</version>
```
Comment (sdevalapurkar-bcgov) [inline src/backend/management-api/pom.xml:106]:
```suggestion
      <version>2.9.2</version> 
```

File: src/backend/management-api/pom.xml [hunk 2]
```diff
@@ -103,12 +103,12 @@
...
       <groupId>io.springfox</groupId>  
       <artifactId>springfox-swagger2</artifactId>  
-      <version>2.9.2</version> 
+      <version>3.0.0</version> 
     </dependency>  
     <dependency> 
       <groupId>io.springfox</groupId>  
       <artifactId>springfox-swagger-ui</artifactId>  
-      <version>2.9.2</version> 
+      <version>3.0.0</version>
```
Comment (sdevalapurkar-bcgov) [inline src/backend/management-api/pom.xml:111]:
```suggestion
      <version>2.9.2</version> 
```

File: src/backend/libs/justin-data-starter/pom.xml [hunk 3]
```diff
@@ -123,7 +123,7 @@
 			<plugin>
 				<groupId>org.codehaus.mojo</groupId>
 				<artifactId>build-helper-maven-plugin</artifactId>
-				<version>1.8</version>
+				<version>3.2.0</version>
```
Comment (sdevalapurkar-bcgov) [inline src/backend/libs/justin-data-starter/pom.xml:126]: this is a major version update, not sure if we want this @TayGov

File: src/backend/libs/justin-data-starter/pom.xml [hunk 4]
```diff
@@ -182,7 +182,7 @@
...
```
Comment (sdevalapurkar-bcgov) [inline src/backend/libs/justin-data-starter/pom.xml:185]: same here

File: src/backend/libs/justin-data-starter/pom.xml (hunk 3 above)
Comment (TayGov) [reply src/backend/libs/justin-data-starter/pom.xml:126]: No that is a huge bump. Might be something we want to test locally first.

File: src/backend/libs/justin-data-starter/pom.xml (hunk 4 above)
Comment (sdevalapurkar-bcgov) [inline src/backend/libs/justin-data-starter/pom.xml:185]:
```suggestion
				<version>1.8</version>
```

File: src/backend/libs/justin-data-starter/pom.xml (hunk 3 above)
Comment (sdevalapurkar-bcgov) [inline src/backend/libs/justin-data-starter/pom.xml:126]:
```suggestion
				<version>1.8</version>
```

**Review by sdevalapurkar-bcgov — COMMENTED** [review]
undoing for time being

**Review by TayGov — APPROVED** [review]

**Pull request merged**